src/ui/common/src/components/integrations/dialogs/s3Dialog.tsx

```diff
@@ -61,4 +98,111 @@ export const S3Dialog: React.FC<Props> = ({ setDialogConfig }) => {
       />
     </Box>
   );
+
+  const configPathTab = (
+    <Box>
+      <Typography variant="body2" color="gray.700">
+        Provide the credentials by specifying the path to your aws credentials,
+        together with the name of the profile you would like to use. <br />
+        The path has to be in the same machine as the one you run{' '}
+        <code> aqueduct start </code> . Typically, it&apos;s{' '}
+        <code>~/.aws/credentials</code>. <br />
+        Once connected, any updates to the file content will automatically apply
+        to this integration.
+      </Typography>
+      <IntegrationTextInputField
+        spellCheck={false}
+        required={true}
+        label="AWS Credentials File Path*"
+        description={'The absolute path to the credentials file'}
+        placeholder={Placeholders.access_key_id}
+        onChange={(event) => setConfigFilePath(event.target.value)}
+        value={accessKeyId}
+      />
+
+      {configProfileInput}
+    </Box>
+  );
+
+  const configUploadTab = (
+    <Box>
+      <Typography variant="body2" color="gray.700">
+        Upload your aws credentials file and provide the name of the profile you
+        would like to use. Typically, it&apos;s <code>~/.aws/credentials</code>.
```

Comment:
Since we want a profile here as well, let's use similar text to what I suggested above.